testing/mongo-c-driver/APKBUILD

```diff
@@ -0,0 +1,61 @@
+# Maintainer: Leonardo Arena <rnalrd@alpinelinux.org>
+pkgname=mongo-c-driver
+pkgver=1.14.0
+pkgrel=0
+pkgdesc="Client library written in C for MongoDB"
+url="https://github.com/mongodb/mongo-c-driver"
+arch="all"
+license="Apache-2.0"
+makedepends="openssl-dev cyrus-sasl-dev icu-dev py3-sphinx cmake"
+subpackages="${pkgname}-static ${pkgname}-dev ${pkgname}-doc libbson"
+source=${pkgname}-${pkgver}.tar.gz::https://github.com/mongodb/${pkgname}/archive/${pkgver}.tar.gz
```

Comment:
```suggestion
source="${pkgname}-${pkgver}.tar.gz::https://github.com/mongodb/${pkgname}/archive/${pkgver}.tar.gz"
```

Reply:
Fixed.
I will mark this conversation as resolved.